adcc/backends/pyscf.py

```diff
@@ -59,28 +59,34 @@ def nabla(self):
 
     @property
     def pe_induction_elec(self):
-        if hasattr(self.scfres, "with_solvent"):
-            if isinstance(self.scfres.with_solvent, pol_embed.PolEmbed):
-                def pe_induction_elec_ao(dm):
-                    return self.scfres.with_solvent._exec_cppe(
-                        dm.to_ndarray(), elec_only=True
-                    )[1]
-                return pe_induction_elec_ao
+        if not hasattr(self.scfres, "with_solvent"):
+            return None
+        if not hasattr(self.scfres.with_solvent, "cppe_state"):
+            return None
```

Comment:
Could also provoke an attributeerror by directly accessing self.scfres.with_solvent.cppe_state and if it is encountered return None.

Reply:
Yes, that's more pythonic 👍

Reply:
Can't do the same for COSMO though since there's no equivalent state object on `with_solvent` in that case + need to make sure we have the correct environment model at that spot.

Reply:
ok